### DISCUSSION TITLE
Testing toyota integration.

### DISCUSSION THREAD

**Discussion by DurgNomis-drol** [DurgNomis-drol/ha_toyota#16] [General]
Hey @CM000n and everyone else! 

I just released a new version of this integration. It is mainly adding some features under the hood to make it more robust and easier to add things in the future.

Can you guys please test it and come with some feedback about it. 😄 🚀 

Remember to remove and re-add it again as the authentication have changed. 

When this is confirmed working i will commence work making tests and applying for this to get added to home assistant core.

**Comment by CM000n** [comment]
Hi @DurgNomis-drol ,  
unfortunately I run into the following error message with the new version and no entities were created

```
Logger: custom_components.toyota
Source: custom_components/toyota/__init__.py:74
Integration: Toyota Connected Services (documentation, issues)
First occurred: 12:54:51 (1 occurrences)
Last logged: 12:54:51

'<' not supported between instances of 'dict' and 'dict'
```

**Reply by DurgNomis-drol** [reply]
Thanks, my mistake. Wrong version of mytoyota was used. Any futher issues, please open a bug report instead. 😄 Should be fixed now in latest master.
